torch/onnx/_patch_torch.py

```diff
@@ -129,7 +129,7 @@ def _new_node(
     Returns:
         The new node.
     """
-    aten = kwargs.pop("aten", False)
+    aten = kwargs.pop("aten", False) or namespace == "aten"
```

Comment:
```suggestion
    aten = kwargs.pop("aten", False) or (namespace == "aten" and op == "ATen")
```

Reply:
I believe the name check is also needed because when `export(..., operator_export_type=ONNX_FALLTHROUGH`), any aten operator might show here, not only aten fallback ones emitted by `_aten_op`

Reply:
When `ONNX_FALLTHROUGH`, would this _new_node function be called? would we just retain the original graph?

Reply:
Is checking `op == "ATen"` enough?